gtfs/spec/en/reference.md

```diff
@@ -29,174 +29,176 @@ This document explains the types of files that comprise a GTFS transit feed and
 
 This section defines terms that are used throughout this document.
 
-* **Field required** - The field column must be included in your feed, and a value must be provided for each record. Some required fields permit an empty string as a value. To enter an empty string, just omit any text between the commas for that field. Note that 0 is interpreted as "a string of value 0", and is not an empty string. Please see the field definition for details.
-* **Field optional** - The field column may be omitted from your feed. If you choose to include an optional column, each record in your feed must have a value for that column. You may include an empty string as a value for records that do not have values for the column. Some optional fields permit an empty string as a value. To enter an empty string, just omit any text between the commas for that field. Note that 0 is interpreted as "a string of value 0", and is not an empty string.
-* **Field conditionally required** - This field or file is **required** under certain conditions, which are outlined in the field or file *Details*. Outside of these conditions, this field or file is optional. 
-* **Dataset unique** - The field contains a value that maps to a single distinct entity within the column. For example, if a route is assigned the ID **1A**, then no other route may use that route ID. However, you may assign the ID **1A** to a location because locations are a different type of entity than routes.
+* **Feed** - Successive datasets comprise a feed.
+* **Dataset** - A complete set of files defined by this specification reference. Datasets should be published at a public, permanent URL, including the zip file name. (e.g., www.agency.org/gtfs/gtfs.zip). For more information see GTFS best practices. 
```

Comment:
Agreed, I'm not sure the "Feed" definition will be universally agreed-upon.  We should probably vote on that separately.

Reply:
I'd expand the example to the full URL - `https://www.agency.org/gtfs/gtfs.zip`

Reply:
I could see how this would be debated, but I also am having a hard time seeing alternative definitions that are usable. The definitions we have proposed are based on existing usage, albeit quite inconsistent usage.

schema.org's uses of the terms "data feed" and "dataset" seem to match what we are doing:
- https://schema.org/DataFeed
- https://schema.org/Dataset

Basically, I think that we should be bold in this case. The terms "dataset" and "feed" have been consistently applied in this proposed spec modification. We could revert, but it hurts to leave this inconsistency in the Spec.

An idea: Prior to the pull request shall we bring the question to a somewhat broader group to see if they agree with our change?